### PR TITLE
feat(devenv): Less strict envrc

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -5,17 +5,11 @@ missing() {
 
 set -e
 
-if missing pre-commit; then
-    echo >&2 "You're missing pre-commit. Install it with brew install pre-commit"
-    return 1
-fi
-# If installed, it executes very fast
-pre-commit install >/dev/null
```
Comment (asottile-sentry) [inline .envrc:13]: we want everyone to use the git hooks

File: .envrc [hunk 2]
```diff
@@ -5,17 +5,11 @@ missing() {
...
 # Convenient if you want secrets to load (e.g. personal token, Github webhook secret) or override values from env.development
 dotenv_if_exists .env
 
 if [ ! -d .venv ]; then
-    python3 -m venv .venv
+    python3.11 -m venv .venv
```
Comment (asottile-sentry) [inline .envrc:12]: this was fine before -- it'll use the correct python due to either pyenv or `devenv`

File: .envrc (hunk 1 above)
Comment (armenzg) [reply .envrc:13]: @asottile-sentry is `brew install pre-commit` the preferred approach?
I would have expected devenv in the sentry's checkout to install it for me.

Comment (asottile-sentry) [reply .envrc:13]: oh I thought this was sentry uhmmm hmmmm

Comment (asottile-sentry) [reply .envrc:13]: I'm not quite sure then

Comment (armenzg) [reply .envrc:13]: Can we remove it for this project?

Comment (asottile-sentry) [reply .envrc:13]: installing it with brew I think is fine -- I would keep this block

**Pull request closed without merging**